packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/tags/GetTagsTests.java

```diff
@@ -226,4 +236,93 @@ public void checkThatDeletedTagIsNotRetrievedAnymore() throws Exception
         returnedCollection.assertThat().entriesListIsNotEmpty()
                 .and().entriesListDoesNotContain("tag", removedTag.toLowerCase());
     }
+
+    /**
+     * Verify if count field is present and with value 0 for searched tag.
+     */
+
+    @Test(groups = { TestGroup.REST_API, TestGroup.TAGS, TestGroup.REGRESSION })
+    public void testGetTag_includingCount()
+    {
+        STEP("Create single tag as admin including count and verify if it is 0");
```

Comment:
We don't need to include the count here (can be fixed in comment and code), and you're not verifying it is zero (can be fixed in comment).

Reply:
Same comment for the next test too.

Reply:
Step description is misleading indeed. Some of it stayed from my initial version. I'll correct that. Thanks